ydb/core/tx/datashard/local_kmeans.cpp

```diff
@@ -544,10 +549,10 @@ class TLocalKMeansScan final: public TLocalKMeansScanBase, private TCalculation<
         return true;
     }
 
-    EScan FeedSample(const TRow& row)
+    EScan FeedSample(TArrayRef<const TCell> row)
     {
-        Y_ASSERT(row.Size() == 1);
-        const auto embedding = row.Get(0).AsRef();
+        Y_ASSERT(row.size() == 1);
+        const auto embedding = row.at(0).AsRef();
```

Comment:
Get was opeator[] btw, but I don't think it's matter, just in case

Reply:
Yeah, just refactored with minimal diff